### PR TITLE
Remove unused imports and function

### PR DESCRIPTION
**Pull request by mcarlton00**
`get_play_action` was only called in this one instance, and would always return 'Play' from my testing.

The imports were called, but never used

### CODE REVIEW

File: resources/lib/objects/__init__.py [hunk 1]
```diff
@@ -6,5 +6,6 @@
 from actions import Actions
 from actions import PlaylistWorker
 from actions import on_play, on_update, special_listener
+import utils
```
[the remark this answers is not in the document]
Comment (mcarlton00) [reply resources/lib/objects/__init__.py]: It does, in fact, use all of the functions in utils.  Because there's exactly one function there, and as far as I can tell, it only gets called once in the entire codebase.  I'm playing with the idea of removing it altogether, but haven't gotten to it yet.

https://github.com/jellyfin/jellyfin-kodi/blob/5890814ae68629042615bbafcfb53968c606e95f/resources/lib/entrypoint/service.py#L79

TLDR: Yes, it could be more specific, but it's likely going away or getting moved in some fashion in the near future anyway

**Comment by TrueTechy** [comment]
Testing shows no issues here. Can login and sync as well as play back content fine
